tutorials/scripting/gdextension/gdextension_file.rst

```diff
@@ -0,0 +1,141 @@
+.. _doc_gdextension_file_sections:
+
+The .gdextension file
+=====================
+
+Introduction
+------------
+
+The ``.gdextension`` file in your project contains the instructions for how to load
+the GDExtension. The instructions are separated into specific sections. This page
+should give you a quick overview of the different options available to you. For an introduction
+how to get started with GDExtensions take a look at the :ref:`GDExtension C++ Example <doc_gdextension_cpp_example>`.
+
+Configuration section
+---------------------
+
++-------------------------------+------------------------------------------------------------------------------------------------------+
+| Instruction                   | Description                                                                                          |
++===============================+======================================================================================================+
+| **entry_symbol**              | Name of the entry function for initializing the GDExtension. This function should be defined in      |
+|                               | the ``register_types.cpp`` file. Adding this is necessary for the extension to work.                 |
++-------------------------------+------------------------------------------------------------------------------------------------------+
+| **compatibility_minimum**     | Minimum compatible version. This prevents older versions of Godot from loading extensions that       |
+|                               | depend on features from newer versions of Godot. **Only supported for Godot 4.1+**                   |
++-------------------------------+------------------------------------------------------------------------------------------------------+
+| **compatibility_maximum**     | Maximum compatible version. This prevents newer versions of Godot from loading the extension.        |
+|                               | **Only supported for Godot 4.3+**                                                                    |
++-------------------------------+------------------------------------------------------------------------------------------------------+
+| **reloadable**                | Reloads the extension upon recompilation. Reloading is not supported for every exposed class.        |
+|                               | This flag should be mainly used for developing or debugging an extension.                            |
++-------------------------------+------------------------------------------------------------------------------------------------------+
+| **android_aar_plugin**        | The GDExtension is part of a :ref:`v2 Android plugin <doc_android_plugin>`. During export this flag  |
```

Comment:
It's not clear from this description that this is a boolean property. Perhaps we should be listing the type for each property?

Reply:
Thanks!